sql/src/main/java/org/apache/druid/sql/calcite/schema/DruidSchema.java

```diff
@@ -305,6 +305,7 @@ public void run()
             }
             catch (InterruptedException e) {
               // Just exit.
+              Thread.currentThread().interrupt();
```

Comment:
Is it aligned with the semantics of "Just exit"? As a minimum, the comment should go after, not before the `Thread.currentThread().interrupt();` statement.